### PR TITLE
DASD-8236 Add Create Namespace Step

### PR DESCRIPTION
**Pull request by smneal**
Also includes script for creating exception

### CODE REVIEW

File: azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    kubernetesCluster: ${{ parameters.AksClusterName }}
+    useClusterAdmin: true
+    command: 'get'
+    arguments: 'namespace'
+- task: PowerShell@2
+  displayName: 'PowerShell: Test-NamespaceExists.ps1'
+  inputs:
+    targetType: 'filePath'
+    ${{ if eq(parameters.IsMultiRepoCheckout, true)  }}:
+      filePath: das-platform-automation/Infrastructure-Scripts/Test-NamespaceExists.ps1
```
Comment (NickGraham101) [inline azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml]: Needs to be updated with new script name `Test-KubernetesNamespaceExists.ps1`

File: azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    ${{ if eq(parameters.IsMultiRepoCheckout, false)  }}:
+      filePath: Infrastructure-Scripts/Test-NamespaceExists.ps1
```
Comment (NickGraham101) [inline azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml]: Needs to be updated with new script name `Test-KubernetesNamespaceExists.ps1`

File: azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml (hunk 1 above)
Comment (smneal) [reply azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml]: These have been updated.

File: azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml (hunk 2 above)
Comment (smneal) [reply azure-pipelines-templates/deploy/step/create-kubernetes-namespace.yml]: These have been updated.

**Review by nbowes24 — APPROVED** [review]

**Pull request merged**